### DISCUSSION TITLE
[Feature request] override legend properties for field

### DISCUSSION THREAD

**Discussion by yangguang760** [grafana/grafana#54399] [Dashboards, Panels & Visualizations]
I want to hide legends for some fields, and found that the legend peoperties could not be overrided.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
